pkg/apis/core/v1alpha1/conversions.go

```diff
@@ -1549,6 +1550,22 @@ func Convert_garden_Shoot_To_v1alpha1_Shoot(in *garden.Shoot, out *Shoot, s conv
 	out.Spec.SecretBindingName = in.Spec.Cloud.SecretBindingRef.Name
 	out.Spec.SeedName = in.Spec.Cloud.Seed
 
+	if email, ok := in.Annotations[constants.AnnotationShootOperatedBy]; ok {
+		if in.Spec.Monitoring.Alerting.EmailReceivers != nil {
+			exists := false
+			for _, receiver := range in.Spec.Monitoring.Alerting.EmailReceivers {
+				if receiver == email {
+					exists = true
+					break
+				}
+			}
+			if !exists {
+				out.Spec.Monitoring.Alerting.EmailReceivers = append(out.Spec.Monitoring.Alerting.EmailReceivers, email)
+			}
+		}
+		delete(out.Annotations, constants.AnnotationShootOperatedBy)
```

Comment:
Shouldn't this only be done if the email was added to the list of receivers in line `1563`?

Reply:
No because if the email isn't added it means it's already in the list and we can remove the annotation since it no longer does anything.

Reply:
OK, I see, thank you!